person.rb

```diff
@@ -1,18 +1,22 @@
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
 class Nameable
   def correct_name
     raise NotImplementedError, 'Subclasses must implement this method'
   end
 end
 
 class Person < Nameable
-  attr_accessor :id, :name, :age
+  attr_reader :rentals
```

Comment:
- [ ] nice work. please ensure Getters and Setters for @name,@age,@rentals should be present
`attr_accessor :name, :age, :rentals`